### ISSUE TITLE
<class 'http.client.IncompleteRead'>

### ISSUE THREAD

**Issue by Khalilsqu** [obspy/obspy#2894]
I keep getting this error when reading IRIS event cataloged "http error IncompleteRead(172226 bytes read)"

How can I resolve it?

```
INFO:root:http error IncompleteRead(172226 bytes read) at 1993-01-01 00:00:00 1994-01-01 00:00:00
INFO:root:Beginning loop 1993-01-01 00:00:00 - 1994-01-01 00:00:00
INFO:root:http error IncompleteRead(172226 bytes read) at 1993-01-01 00:00:00 1994-01-01 00:00:00
INFO:root:Beginning loop 1993-01-01 00:00:00 - 1994-01-01 00:00:00
INFO:root:http error IncompleteRead(172226 bytes read) at 1993-01-01 00:00:00 1994-01-01 00:00:00
INFO:root:Beginning loop 1993-01-01 00:00:00 - 1994-01-01 00:00:00
INFO:root:http error IncompleteRead(172226 bytes read) at 1993-01-01 00:00:00 1994-01-01 00:00:00
INFO:root:Beginning loop 1993-01-01 00:00:00 - 1994-01-01 00:00:00
INFO:root:http error IncompleteRead(172226 bytes read) at 1993-01-01 00:00:00 1994-01-01 00:00:00
INFO:root:Beginning loop 1993-01-01 00:00:00 - 1994-01-01 00:00:00
INFO:root:http error IncompleteRead(172226 bytes read) at 1993-01-01 00:00:00 1994-01-01 00:00:00
INFO:root:Beginning loop 1993-01-01 00:00:00 - 1994-01-01 00:00:00
INFO:root:Beginning loop 1994-01-01 00:00:00 - 1995-01-01 00:00:00
INFO:root:Beginning loop 1995-01-01 00:00:00 - 1996-01-01 00:00:00
```

**Comment by d-chambers** [comment]
Please include enough to reproduce the issue, including code example or file attachments

**Comment by Khalilsqu** [comment]
[Here](https://github.com/obspy/obspy/issues/2895#issue-1045199953)you will find a full reproducible code

**Comment by d-chambers** [comment]
Ok, I think this was resolved in #2895. Feel to reopen if not.
